pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
Summary:

1.  (discussed in chat): test running as UID 0 without privileged
    1.  if you don't need privileged you should also test running as UID 0 AND an SELinux context
1.  if you can run as root with an SELinux context add the SA to the `anyuid` SCC
1.  if you can run as root but not with the SELinux context add the SA to `privileged` and use `RunAsUser: 0` in the SC
1.  if you need privileged add the SA to `privileged` and use `Privileged: true`
1.  remove the SA from the `hostmount` SCC

Did I miss anything?